DonjinKrawler-Client/src/main/java/donjinkrawler/packetcontrol/handlers/RoomPacketHandler.java

```diff
@@ -0,0 +1,17 @@
+package donjinkrawler.packetcontrol.handlers;
```

Comment:
"Use Intention-Revealing Names"
Why is the client package 'packagecontrol' and the server package 'pckcontrol'?
Most, if not all, the class names are the same, we could be better off with a distinction between them.
I was stumped when looking for a RoomPacketHandler which had this code but I was looking at the server one.